syft/codes.py

```diff
@@ -16,6 +16,13 @@ class PLAN_CMDS(object):
     FETCH_PROTOCOL = "fetch_protocol"
 
 
+class TENSOR_SERIALIZATION(object):
+    TORCH = "torch"
```

Comment:
In the future we need to turn these to be integer to have faster serde, but maybe for debug it's convenient to keep it like this